Cargo.lock

```diff
@@ -6724,6 +6724,7 @@ dependencies = [
  "solana-measure",
  "solana-metrics",
  "solana-sdk",
+ "solana-vote",
```

Comment:
Same comment about pulling in `crossbeam-channel` to `solana-program-runtime`

Reply:
Rebased on top of #1623.